lib/kubeclient/common.rb

```diff
@@ -109,6 +109,10 @@ def self.define_entity_methods(entity_types)
         define_method("update_#{entity_name}") do |entity_config|
           update_entity(entity_type, entity_config)
         end
+
+        define_method("patch_#{entity_name}") do |name, namespace = nil, patch|
```

Comment:
@moolitayer I think you can still call ('somename', {somevalues}) it worked for me